components/Filters/ActionsFilterToolbar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { c } from 'ttag';
+import { Button, PrimaryButton, useModals } from 'react-components';
+
+import AddFilterModal from '../../containers/Filters/AddFilterModal';
+
+function ActionsFilterToolbar() {
+    const { createModal } = useModals();
+
+    const handleClickAdd = (type) => () => {
+        createModal(<AddFilterModal type={type} />);
+    };
+
+    return (
+        <>
+            <PrimaryButton onClick={handleClickAdd()}>{c('Action').t`Add Filter`}</PrimaryButton>
+            <Button onClick={handleClickAdd('complex')} className="ml1">
```

Comment:
As a good practice use `mr1` on the first button than the second button.

Reply:
>  The second element should care about the previous one.

yep that's why there is the className on it ;) cf the adjacent selector